syntax = "proto2";
package phxbinlog;

message UserInfo
{
	required string username = 1;
	required string pwd = 2;
}

message MasterInfo
{
//	required string ip = 1;
	required uint32 port = 2;
	optional uint64 version = 3;
	optional uint32 svr_id = 4;
	optional uint32 update_time = 5;
	optional uint32 expire_time = 6;
	optional uint32 lease = 7;
	optional uint64 instance_id = 8;
	optional string export_ip = 9;
	optional uint32 export_port = 10;
	repeated UserInfo admin_userinfo = 11;
	repeated UserInfo replica_userinfo = 12;
};

